### ISSUE TITLE
Can't get a gold ring from bartering with piglins

### ISSUE THREAD

**Issue by benkaminiscool** [Additional-Mods/additionaladditions#122]
I've tried, and no matter what, piglins won't give me a gold ring. I tested this in a creative world, and it didn't work. Also, on my world I have a mod that adds more advancements to the game (It's blazeandcave's advancements pack) which ended up giving me an advancement for getting all possible piglin trades. However, I didn't get a ring. If there is a way to craft this or something like that, please let me know. Thank you!

**Comment by Dr-Ner0** [comment]
Yeah, same problem here, also it says there should be option to enable or disable golden rings in the config file but it must be missing cus I cannot find any mention about golden rings in the config file

**Comment by kurobara31694** [comment]
In creative, I used stack and stacks of gold blocks worth of gold to trade and nothing.....
